doc/EnvironmentVariables.md

```diff
@@ -7,6 +7,7 @@
 | ConfigRefreshFrequencySecs           | Interval at which the EdgeAgent config is refreshed from upstream                                                               | int32                                         | 3600                       |
 | ConfigSource                         | twin / local, specifies where the deployment config should be read from                                                         | twin, local                                   | twin                       |
 | CoolOffTimeUnitInSeconds             | Time span to wait between restart attempts on a module                                                                          | 0-300                                         | 10                         |
+| DisableDeviceAnalyticsMetadata       | Whether to disable sending basic metadata about the device to Microsoft.                                                        | bool                                          | false                      |
```

Comment:
I think we should mention that the metadata is largely identical to the customer-facing metrics.

Reply:
...or just link to https://github.com/Azure/azure-iotedge/blob/main/README.md#telemetry?